### DISCUSSION TITLE
iBiblia v1.0.0.68

### DISCUSSION THREAD

**Discussion by SalitaNgDiyos** [rubiot/ibiblia#100] [General]
Review or Feedback about this new release

https://github.com/rubiot/ibiblia/releases/tag/v.1.0.0.68

**Comment by SalitaNgDiyos** [comment]
Thanks brother, for this new release!

My initial observations:

1. Exporting a bible module is so much faster. 10x as fast as the previous versions.
2. No longer need to use a file syncing app just to automate copying exported files to current Word folders. I can now export directly to the Word program bible folder!
3. Print book chapter is cool to have without going to theWord app

**Comment by jonathangkoehn** [comment]
I noticed a speed improvement as well. (I also installed to a SSD so that helped as well.)
Thank you for all your hard work on this Rubio. It is a helpful way for me to work on my own translation/studies as I prepare for sermons and study in general.

**Reply by SalitaNgDiyos** [reply]
I should say that you will be greatly blessed in your spiritual life as you endeavor to work on your own translation of the Scriptures. It is indeed the BEST way to know the God of the Bible. The translation and the sermons are only the byproducts of this work not the goal. 

Edmund

**Reply by jonathangkoehn** [reply]
Agreed, the goal is to know God. I am thankful for the tools God has given that such as iBiblia and theWord. 
Not sure when I began translating but many years. I came out of a very legalistic opinionated religion and want to not know opinons/rules but instead to know Jesus Christ.

What have you found that has encouraged you as you have been translating?

**Reply by SalitaNgDiyos** [reply]
I came from a Pentecostal, Prosperity Gospel background and became a minister in that kind of environment. When I left there, I started to translate in our mother tongue after realizing that many of my fellow Filipinos living outside the Capital City do not have access to hard copy Bibles. Our Bible Society is run by Roman Catholics; so they are sometimes hard to understand in many passages. 

I use the translation method done with the KJV, "as literal as possible" after I realized the error of the dynamic method of translation. I thought that it was the better method, but as I kept moving deep in the work, I realized that words can only be fully understood based on their context, not in the Greek lexicons. 

Since I started in the New Testament, I came to know the true Christ of the Gospels through His guidance; that he was not as the Prosperity Gospel Preachers are teaching that He is. Translating taught me to treat the Word as they should be treated and developed in me a keen eye on every word of Scripture by reading at a slow pace. 

So far, the Gospel books I've produced are currently being read in the homes of the people I disciple. I love using them more than any other translation because I understood them...If we understand the Bible we are teaching, it gives us the ability to teach it with power just as Christ and the apostles did. The effectiveness is tremendous. There is no more dark areas in Scriptures, and the time it takes to prepare what to teach goes to as minimal as zero, because I've been through the Word. I just need to gaze on the pages of the Gospels and instantly I know what to say without hesitation.

**Comment by SalitaNgDiyos** [comment]
Sync'ing strongs/morph on auto-select causes a 1 sec delay in associating words. Hope we can toggle this feature in the future release.

**Reply by SalitaNgDiyos** [reply]
I just upgraded my entire PC to AMD Ryzen 5600G DDR4 16GB RAM from AMD A8 7800 DDR3 16GB RAM and I noticed that all the lags were gone!

**Comment by SalitaNgDiyos** [comment]
Anti Virus Alert on iBiblia.exe

I am using Windows 10 Pro for years without a 3rd party anti virus. So I think there is no issue with iBiblia.exe for that reason. Hehe
